File: src/com/topface/topface/ui/MainActivity.java
package com.topface.topface.ui;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import com.topface.topface.Data;
import com.topface.topface.R;
import com.topface.topface.utils.Debug;

public class MainActivity extends Activity {
    // Data

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Debug.log(this, "+onCreate");
        setContentView(R.layout.ac_main);

        //startService(new Intent(getApplicationContext(), ConnectionService.class));

        if (Data.isSSID())
            startActivity(new Intent(getApplicationContext(), DashboardActivity.class));
        else
            startActivity(new Intent(getApplicationContext(), AuthActivity.class));

        finish();
    }

    @Override
    protected void onDestroy() {
        Debug.log(this, "-onDestroy");
        super.onDestroy();
    }

}

File: libs/MyApplication/app/src/main/java/com/example/ppavlik/myapplication/MainActivity.kt
package com.example.ppavlik.myapplication

import android.databinding.DataBindingUtil
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.example.ppavlik.myapplication.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding: ActivityMainBinding = DataBindingUtil.setContentView(this, R.layout.activity_main)
        binding.dataTest = DataTest()
    }
}